<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparasionWindow</class>
 <widget class="QWidget" name="ComparasionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comparator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>First triangle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Side a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideALabel"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Side b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideBLabel"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Side c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideCLabel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="signLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Second triangle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Side a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideA2Label"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Side b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideB2Label"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Side c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideC2Label"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="compareButton">
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
